src/main/java/org/entando/plugins/pda/pam/service/task/util/TaskUtil.java

```diff
@@ -21,13 +19,16 @@
     @SuppressWarnings("unchecked")
     public static Map<String, Object> flatProperties(Map<String, Object> data) {
         try {
-            String flatten = JsonFlattener.flatten(OBJECT_MAPPER.writeValueAsString(
-                    Optional.ofNullable(data).orElse(new HashMap<>())));
+            if (data == null || data.isEmpty()) {
+                return null;
```

Comment:
Why are we returning null on this method? I believe the previous implementation is better because we don't need to handle nullability on the caller.

Reply:
> Why are we returning null on this method? I believe the previous implementation is better because we don't need to handle nullability on the caller.

This is not present on final version o PR.